File: Selenium/Activities/Activity19.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity19 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/javascript-alerts");
		System.out.println("The Title of the page is  : " + driver.getTitle());

		driver.findElement(By.id("simple")).click();

		Alert alt = driver.switchTo().alert();

		String altText = alt.getText();

		System.out.println("The Alert Text is : " + altText);
		alt.accept();

		driver.findElement(By.id("confirm")).click();

		driver.switchTo().alert().dismiss();

		driver.quit();

	}

}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("title is : ", driver.title)

    driver.find_element(By.ID, "confirm").click()
    alert =driver.switch_to.alert

    print("the alert text is : " ,alert.text)
    alert.accept()

    driver.find_element(By.ID, "confirm").click()
    driver.switch_to.alert.dismiss()

    driver.quit()

